css/style.css

```diff
@@ -1,61 +1,224 @@
+/* Mobile-first redesign - and Media query for Homework requirements of Desktop */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  overflow-x: hidden;
+}
+
+h1 {
+  font-family: "Vollkorn", serif;
+  font-weight: 500;
+  font-size: 2em;
+}
+
+.container {
+  margin: 0 auto;
+  padding: 40px;
+  width: 100%;
+}
+
+header {
+  display: flex;
+}
+
 .banner {
-width: 1600px;
-height: 392px;
-background: #000000;
-opacity: 0.3;
-background-image: url(/images/banner\ image.jpg);
-}
-
-.titlebanner {
-width: 566px;
-height: 42px;
-font-style: normal;
-font-weight: 500;
-font-size: 36px;
-line-height: 42px;
-color: #FFFFFF;
-}
-
-.btn{
-left: 24.23%;
-right: 24.62%;
-top: 33.33%;
-bottom: 32.1%;
-font-style: normal;
-font-weight: 500;
-font-size: 24px;
-line-height: 28px;
-color: #FFFFFF;
-}
-
-.title {
-font-style: normal;
-font-weight: bold;
-font-size: 18px;
-line-height: 21px;
-/* identical to box height */
-color: #000000;
-}
-
-.author{
-font-style: italic;
-font-weight: normal;
-font-size: 18px;
-line-height: 21px;
-/* identical to box height */
-color: #000000;
+  color: #fff;
+  position: relative;
+  margin: 0 auto 120px;
+}
+
+.banner::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 392px;
+  background: url("../images/banner-image.jpg") no-repeat top center;
+  -webkit-filter: brightness(0.7);
+  filter: brightness(0.7);
+  z-index: -1;
+}
+
+.banner-text {
+  padding: 5% 0;
+  width: 100%;
+  height: 300px;
+}
+
+.banner-text h1 {
+  font-size: 2em;
+  font-weight: 400;
+  margin-bottom: 40px;
+  margin-top: 60px;
+}
+
+.banner-text button {
+  background-color: transparent;
+  border: 2px solid #fff;
+  padding: 8px 40px;
+  color: #fff;
+  font-size: 1.1em;
+  text-transform: uppercase;
 }
 
 .books {
- display: flex;
- flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  column-gap: 10%;
+  row-gap: 10%;
 }
- 
+
 .book {
-    width: 20%;
-    /* box-sizing: border-box; */
-    margin: 0 20px;
+  height: fit-content;
+}
+
+.book .title {
+  font-size: 1.1em;
+  font-weight: 700;
+  margin-bottom: 10px;
+}
+
+.book .author {
+  font-size: 1.1em;
+  font-weight: 400;
+}
+
```

Comment:
We never mentioned that responsiveness should be a part of the task.
You may delete this part.

Reply:
Ok I reverted back to Desktop only design , I have a backup of the media queries in case we needed that later on.